Datadog.Trace.sln

```diff
@@ -635,8 +637,8 @@ Global
 		{4B243CF1-4269-45C6-A238-1A9BFA58B8CC}.Release|x64.Build.0 = Release|Any CPU
 		{4B243CF1-4269-45C6-A238-1A9BFA58B8CC}.Release|x86.ActiveCfg = Release|Any CPU
 		{4B243CF1-4269-45C6-A238-1A9BFA58B8CC}.Release|x86.Build.0 = Release|Any CPU
-		{0D546118-B70A-44D0-B675-39EDB99FCEEE}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{0D546118-B70A-44D0-B675-39EDB99FCEEE}.Debug|Any CPU.Build.0 = Debug|x86
+		{0D546118-B70A-44D0-B675-39EDB99FCEEE}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{0D546118-B70A-44D0-B675-39EDB99FCEEE}.Debug|Any CPU.Build.0 = Debug|x64
```

Comment:
{0D546118-B70A-44D0-B675-39EDB99FCEEE} is Datadog.Trace.ClrProfiler.IntegrationTests.csproj, Any CPU needs to be either x86 or x64, so it knows what version of the test to run, I think x64 makes more sense as a default.